lib/src/json_schema/validator.dart

```diff
@@ -45,11 +45,42 @@ import 'package:json_schema/src/json_schema/schema_type.dart';
 import 'package:json_schema/src/json_schema/utils.dart';
 import 'package:json_schema/src/json_schema/global_platform_functions.dart' show defaultValidators;
 
+class Instance {
+  Instance(dynamic data, {String path = ''}) {
+    this.data = data;
+    this.path = path;
+  }
+
+  dynamic data;
+  String path;
+
+  @override
+  toString() => data.toString();
+}
+
+class ValidationError {
```

Comment:
did we export this class for consumers?

Reply:
This is not intended for consumers to instantiate directly, but they are supposed to have read access via the API in the `JsonSchema` object.